pwiz_tools/Shared/ProteowizardWrapper/ProteowizardWrapper.csproj

```diff
@@ -112,6 +112,10 @@
     </NativeReference>
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\Common\Common.csproj">

```

Comment:
I do not think that this reference needs to be here since "DiffAttribute" is in the "CommonUtil" project not "Common".